js/ui/speed_dial_action/speed_dial_main_item.js

```diff
@@ -195,7 +214,7 @@ const SpeedDialMainItem = SpeedDialItem.inherit({
     _optionChanged(args) {
         switch(args.name) {
             case "actions":
-                this._renderIcon();
+                if(this._isVisible()) this._renderIcon();
```

Comment:
It looks strange that we do not render an icon for an invisible button, but render close icon, set position, etc.

Reply:
It is needed for icon and label because they rerender early then FAB hide. There is this condition in set position too because of the show method.